node/pkg/websocketfetcher/providers/kucoin/kucoin.go

```diff
@@ -15,6 +16,8 @@ import (
 
 type KucoinFetcher common.Fetcher
 
+var pingInterval = 18000
```

Comment:
since this number is being used below as well, how about having it as a constant value somewhere? will make it safer to update in the future